### ISSUE TITLE
[Bug report] Can't enter wind shrine

### ISSUE THREAD

**Issue by Buck4219587** [Pyrdacor/Ambermoon.net#355]
Version 1.8.7, Windows, German 1.18

I was unable to enter the wind shrine. The door screen triggered, I was able to use the shrine key, after that I'm at the main map again at the location of the door (the character is hidden by the door). I can walk one step further into the mountain, but nothing I do triggers a transition to the wind shrine map.

I copied the save to the english version, but the problem was the same here.

(I could work around this by activating the cheats and teleporting to map 408)

**Comment by Pyrdacor** [comment]
Can you send me the save please? Or just upload it here.

Did you walk on the door at first time or access it by dismounting the eagle or broom?

**Comment by Buck4219587** [comment]
Certainly. Here are two saves in front of the shrine, the first one (12) with the door still locked, the second one (13) with the door opened (in the second save I've already used the teleport cheat to produce the necklace).

I dismounted the broom about two tiles south of the door and walked up to it.

[windshrine_save.zip](https://github.com/Pyrdacor/Ambermoon.net/files/11715732/windshrine_save.zip)
